docs/examples/800_istio.adoc

```diff
@@ -0,0 +1,169 @@
+= Istio Support
+
+== Overview
+
+Coherence Operator 3.1.15 and later supports Istio 1.9.1 and later. You can run the operator and Coherence cluster managed by the operator with Istio sidecar injection enabled.  Coherence caches can be accessed from outside via Coherence*Extend, REST, and other supported Coherence clients.  
+
+== Limitations
+
+The current support for Istio has the following limitation:
+
+ * TLS between cluster members do not work in Istio environment. Coherence client can support TLS through Istio secure gateway with TLS termination to connect to Coherence cluster running inside kubernetes.  For example, you can apply the following Istio gateway and and virtual service in the namespace of the Coherence cluster:
```

Comment:
If Coherence is configured with specific cluster ports (like you have to do when using a firewall) and then those ports are exposed in the ports list for the Pod then Istio will intercept those ports. Does this break Coherence? If it does that should be listed as the limitation not TLS. The fact that TLS does not work probably does not then need to be called out if cluster traffic does not work anyway.

It is a bit confusing in the section on limitations then showing how to do TLS for things like Extend, REST etc. How to use TLS like this should be a separate part of the "How To" section below.

Reply:
If Coherence is configured with specific cluster ports (like you have to do when using a firewall) and then those ports are exposed in the ports list for the Pod then Istio will intercept those ports. Does this break Coherence? Yes it breaks Coherence.